src/elfpy/simulators.py

```diff
@@ -236,22 +239,43 @@ def setup_simulated_entities(self, override_dict=None):
         )
         if self.config.simulator.verbose:
             print(
-                f"\n-----\ninitial market values:"
-                f"\nshare_reserves = {self.market.share_reserves}"
-                f"\nbond_reserves = {self.market.bond_reserves}"
-                f"\ntarget_liquidity = {self.target_liquidity}"
-                f"\ntotal market liquidity = {self.market.share_reserves + self.market.bond_reserves}"
-                f"\nfee_percent = {self.market.fee_percent}"
-                f"\nshare_price = {self.market.share_price}"
-                f"\ninit_share_price = {self.market.init_share_price}"
-                f"\ninit_time_stretch = {self.market.time_stretch_constant}"
-                "\n-----\n"
+                f"initial market values:"
+                f"\n target_liquidity = {self.config.simulator.target_liquidity:,.0f}"
+                f"\n init_base_asset_reserves = {init_base_asset_reserves:,.0f}"
+                f"\n init_token_asset_reserves = {init_token_asset_reserves:,.0f}"
+                f"\n init_pool_apy = {self.config.simulator.init_pool_apy:.2%}"
+                f"\n vault_apy = {self.config.simulator.vault_apy[0]:.2%}"
+                f"\n fee_percent = {self.market.fee_percent}"
+                f"\n share_price = {self.market.share_price}"
+                f"\n init_share_price = {self.market.init_share_price}"
+                f"\n init_time_stretch = {self.market.time_stretch_constant}"
+            )
+            print(f"{self.market.get_market_step_string()}")
+        self.block_number = 0
+        print(f" should I initialize the market with a user? init_LP = {self.config.simulator.init_LP}")
+        if self.config.simulator.init_LP:
```

Comment:
Is the init_LP flag meant to allow support for backwards compatibility with the old method of initializing the market? I don't think we really need to support this. Both pricing models should be compatible with the new setup.